### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ScrollWorks**

### CODE REVIEW

**Comment by kyldvs** [comment]
I actually prefer the old terminology using `dispatch` as a verb. It ties the purpose of the Dispatcher to a common English verb in a real scenario. It also keeps the material shorter.

Can you provide a bit more context on why you prefer your phrasing? Happy to discuss

**Comment by ScrollWorks** [comment]
Hey Kyle, thanks for taking the time to reply.

The reason I sent my pull request is because  of a question a colleague asked me while I was trying to explain the basics of flux to him. We read through the page, and when we got to this point in the documentation, while I had no problems with the sentence, my colleague asked: "Shouldn't be the dispatcher the one who notifies the stores subscribed to it?"

I think for someone who is familiar with flux, your phrasing makes perfect sense, and it has the extra benefits you mention in your comment. However, when someone who is new to flux (arguably most people reading this page in the docs) comes across the original sentence, they could interpret that it's the view itself which is dispatching the action and notifying the stores, like it happened to my colleague. 

I think my edit, though a bit more verbose, reflects the data flow more explicitly, and avoids the possibility of making wrong assumptions.

Having said that, neither my friend nor I are native English speakers (we both speak it very fluently though!).

**Comment by kyldvs** [comment]
Thanks for the context. How about this:

> When a user types in the title of a new Todo and hits enter the view tells the Dispatcher to dispatch an action notifying all stores.

**Comment by ScrollWorks** [comment]
Yeah, the two bullets work :)

**Comment by kyldvs** [comment]
Feel free to update and I'll merge it!

File: examples/flux-concepts/README.md [hunk 1]
```diff
@@ -88,11 +88,11 @@ Example:
 1. The main view subscribes to the TodoStore.
 2. It accesses a list of the Todos and renders them in a readable format for
    the user to interact with.
-3. When a user types in the title of a new Todo and hits enter the view will
-   dispatch an action notifying all stores.
-4. The TodoStore handles the action and adds another Todo to its internal
+3. When a user types in the title of a new Todo and hits enter the view tells the Dispatcher to dispatch an action.
```
Comment (kyldvs) [inline examples/flux-concepts/README.md]: Nit: do you mind keeping the source wrapped at 80 columns?

**Comment by kyldvs** [comment]
Thanks!

**Pull request merged**